frontend/src/pages/PlannerWeightsPage.module.scss

```
@@ -2,20 +2,19 @@
 
 .weightsHeader {
   @extend .layout-row;
-  margin-left: 25px;
+  margin: 0 25px;
   align-items: flex-start;
 
   .title {
     @extend .layout-row;
     gap: 12px;
-    align-items: center;
+    align-items: flex-start;
+    margin-right: 25px;
   }
 
   .clientShares {
     display: flex;
-    flex-grow: 1;
-    justify-content: flex-end;
-    margin-right: 25px;
+    margin: 0 34px 0 auto;
```

Now that `margin-right: 25px` is added to the .title, this has some unnecessary margin
```suggestion
    margin-left: auto;
```

---

If there's not margin on the right, element overflows and part of it is hidden, the 34px is (padding 16px + border 1px)*2

---

I can't seem to replicate the overflowing, would you mind showing how to? (on slack or here)